rules-threat-hunting/windows/process_creation/proc_creation_win_webdav_process_execution.yml

```
@@ -0,0 +1,30 @@
+title: Process Execution From WebDAV Share
+id: f8de9dd5-7a63-4cfd-9d0c-ae124878b5a9
+status: experimental
+description: |
+    Detects execution of processes with image paths starting with WebDAV shares (\\), which might indicate malicious file execution from remote web shares.
+    Execution of processes from WebDAV shares can be a sign of lateral movement or exploitation attempts, especially if the process is not a known legitimate application.
+    Exploitation Attempt of vulnerabilities like CVE-2025-33053 also involves executing processes from WebDAV paths.
+references:
+    - https://research.checkpoint.com/2025/stealth-falcon-zero-day/
+author: Swachchhanda Shrawan Poudel (Nextron Systems)
+date: 2025-06-13
+tags:
+    - attack.execution
+    - attack.lateral-movement
+    - attack.t1105
+    - detection.threat-hunting
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Image|startswith: '\\\\'
+        - Image|contains: '\DavWWWRoot\'
+    filter:
+        Image|contains: 'C:\'
+    condition: selection and not filter
```

```suggestion
detection:
    selection:
        Image|startswith: '\\\\'
        Image|contains: '\DavWWWRoot\'
    condition: selection
```
The title says "from webdav share" which is not represented in the detection logic if we use OR. So either update the description/title or the detection logic accordingly.